tests/nat/nat_helpers.py

```diff
@@ -476,7 +476,10 @@ def setup_ptf_interfaces(testbed, ptfhost, duthost, setup_info, interface_type,
     if "dut_iface" in setup_info[interface_type]["vrf_conf"][key].keys():
         dut_iface = setup_info[interface_type]["vrf_conf"][key]["dut_iface"]
         pch_ip = setup_info["pch_ips"][dut_iface]
-        duthost.shell("sudo config interface ip remove {} {}/31".format(dut_iface, pch_ip))
+        pch_mask = setup_info["pch_masks"][dut_iface]
+        mask_prefix = IPAddress(pch_mask).netmask_bits()
```

Comment:
You forgot to import: 
`from netaddr import IPAddress`
Or I miss something?

Reply:
thanks